templates/js-cypress/cypress/e2e/first-spec.cy.js

```diff
@@ -0,0 +1,7 @@
+describe('first tests suite', () => {
+    it('visit Apify main page', () => {
+        cy.visit('https://apify.com');
```

Comment:
I'd make the baseUrl part of the input and pass it via config, so you would only use relative links in visit.

Reply:
Hmm, just thinking about usecase when I would like to upload more tests with different `baseUrls` and run them at once? Then, I would need to separate them and create tasks and adjust the test itself.. 

But if you think that it is edge case and users usually go with one base url, then it makes sense and I would adjust it 👍

Reply:
added to input schema as required field with pre-fill